### PR TITLE
DOC updated the docs on the bug release

### PR DESCRIPTION
**Pull request by maikia**
Updates a few steps for the bug release in the docs

### CODE REVIEW

File: doc/contribute.rst [hunk 1]
```diff
@@ -320,29 +320,29 @@ Patch/bug fix release process
 
 1. Checkout the tag for the last minor release, e.g.,
    `git checkout 0.5.X`.
-   note: X is part of the name, not a number to substitute.
-   note2: you can type `git fetch --all --tags` to fetch available tags.
+   **note**: X is part of the name, not a number to substitute.
+   **note2**: you can type `git fetch --all --tags` to fetch available tags.
 2. There are two ways to continue. If you only want to release a couple of
    commits we recommend that you go with '3a', otherwise the easier will be
```
Comment (lucyleeow) [inline doc/contribute.rst:326]:
```suggestion
   commits we recommend that you go with '3a', if you want to release many commits '3b' would be easier.
```

File: doc/contribute.rst [hunk 2]
```diff
@@ -320,29 +320,29 @@ Patch/bug fix release process
...
+3a: release only a few commits
 
 - Find the commit(s) hash of the bug fix commit you wish to back port using
   `git log`.
 - Append the bug fix commit(s) to the branch using `git cherry-pick <hash>`.
 
-**3b: release multiple commits**
+3b: release multiple commits
 
 - being on the `0.5.X` checkout a branch:
```
Comment (lucyleeow) [inline doc/contribute.rst:337]:
```suggestion
- checkout the `0.5.X` branch:
```

File: doc/contribute.rst [hunk 3]
```diff
@@ -320,29 +320,29 @@ Patch/bug fix release process
...
-  `git checkout -b release-0.5.no master` where 'no' is the number on your bug
-  release.
+  `git checkout -b release-0.5.<number> master` where 'number' is the number of
+  your bug release.
 - commit your desired changes by typing: `git rebase -i master`.
   You should now see the list of all the commits since the last release.
   Indicate which commits you want to release by typing 'pick' on their lefthand
   side, and 'drop' if you do not want to keep it. Make sure to drop the commit
   with previous bumpversion. It's usefult to have a copy of the `git rebase -i`
```
Comment (lucyleeow) [inline doc/contribute.rst:344]:
```suggestion
  with previous bumpversion. It's useful to have a copy of the `git rebase -i`
```

File: doc/contribute.rst [hunk 4]
```diff
@@ -320,29 +320,29 @@ Patch/bug fix release process
...
-  log in the PR to PR to help others understand what's included.
+  log in the PR to help others understand what's included.
 - make a PR to the master. Once it is merged continue to the next step.
```
Comment (lucyleeow) [inline doc/contribute.rst:346]: Do you PR to master or the release branch? Also if you need to make PR here do you also need to make PR at the end of 3a?

File: doc/contribute.rst [hunk 5]
```diff
@@ -312,17 +312,39 @@ The following explain the main steps to release `ramp-board`:
...
+- checkout the `0.5.X` branch:
+  `git checkout -b release-0.5.<number> master` where 'number' is the number of
+  your bug release.
+- commit your desired changes by typing: `git rebase -i master`.
+  You should now see the list of all the commits since the last release.
+  Indicate which commits you want to release by typing 'pick' on their lefthand
+  side, and 'drop' if you do not want to keep it. Make sure to drop the commit
+  with previous bumpversion. It's useful to have a copy of the `git rebase -i`
+  log in the PR to help others understand what's included.
+- make a PR to the master. Once it is merged continue to the next step.
```
Comment (rth) [inline doc/contribute.rst:346]: > Do you PR to master or the release branch?

I also think it was meant to be release

```suggestion
- make a PR to the release branch. Once it is merged continue to the next step.
```

> Also if you need to make PR here do you also need to make PR at the end of 3a?

I guess it depends on the person doing it. If it's just a commit, cherry-picking directly to the release branch sounds also fine. A PR is mostly useful to discuss what to pick or exclude.

File: doc/contribute.rst [hunk 6]
```diff
@@ -312,17 +312,39 @@ The following explain the main steps to release `ramp-board`:
...
-   using `git log`.
-2. Checkout the branch for the lastest release, e.g.,
+1. Checkout the tag for the last minor release, e.g.,
    `git checkout 0.5.X`.
-3. Append the bug fix commit(s) to the branch using `git cherry-pick <hash>`.
+   **note**: X is part of the name, not a number to substitute.
+   **note2**: you can type `git fetch --all --tags` to fetch available tags.
+2. There are two ways to continue. If you only want to release a couple of
+   commits we recommend that you go with '3a', if you want to release many commits '3b' would be easier.
+   to continue with '3b'.
```
Comment (rth) [inline doc/contribute.rst:327]:
```suggestion
   commits we recommend that you go with '3a', if you want to release many commits '3b'
   would be easier.
```

**Review by rth — APPROVED** [review]
Thanks @maikia and @lucyleeow ! I added a couple more changes otherwise LGTM.

**Pull request merged**